docs/source/reference/ndimage.rst

```diff
@@ -27,3 +27,20 @@ OpenCV mode
 -----------
 :mod:`cupyx.scipy.ndimage` supports additional mode, ``opencv``.
 If it is given, the function performs like `cv2.warpAffine <https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga0203d9ee5fcd28d40dbc4a1ea4451983>`_ or `cv2.resize <https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga47a974309e9102f5f08231edc7e7529d>`_.
+
+Example to resize image to half size:
+```
+import cupyx.scipy.ndimage
+import cupy as cp
+import cv2
+
+im = cv2.imread('YOUR_IMAGE_PATH')
+
+M = cp.eye(4)
+M[0][0] = M[1][1] = 0.5
+
+smaller_shape = (im.shape[0]/2, im.shape[1]/2, 3)
+smaller = cp.zeros(smaller_shape) # preallocate memory for resized image
+smaller = cupyx.scipy.ndimage.affine_transform(im, M, output_shape=smaller_shape, output=smaller, mode='opencv')
```

Comment:
I suppose we don't need to assign the returned value to `smaller` as it is passed to `output` parameter.

```suggestion
cupyx.scipy.ndimage.affine_transform(im, M, output_shape=smaller_shape, output=smaller, mode='opencv')
```